code/game/objects/items/devices/portable_vendor.dm

```diff
@@ -239,5 +239,7 @@
 							list("Neurostimulator Implant", 30, /obj/item/implanter/neurostim, "white", "Implant which regulates nociception and sensory function. Benefits include pain reduction, improved balance, and improved resistance to overstimulation and disoritentation. To encourage compliance, negative stimulus is applied if the implant hears a (non-radio) spoken codephrase. Implant will be degraded by the body's immune system over time, and thus malfunction with gradually increasing frequency. Personal use not recommended."),
 							list("Ultrazine Pills", 20, /obj/item/storage/pill_bottle/restricted/ultrazine, "white", "Highly-addictive stimulant. Enhances short-term physical performance, particularly running speed. Effects last approximately 10 minutes per pill. More than two pills at a time will result in overdose. Withdrawal causes extreme discomfort and hallucinations. Long-term use results in halluciations and organ failure. Conditional distribution secures subject compliance. Not for personal use."),
 							list("Cash", 5, /obj/item/spacecash/c1000, "white", "$1000 USD, unmarked bills"),
-							list("Cigars", 5, /obj/item/storage/fancy/cigar, "white", "Case of premium cigars, untampered.")
+							list("Cigars", 5, /obj/item/storage/fancy/cigar, "white", "Case of premium cigars, untampered."),
+							list("De Void of Soul", 51, /obj/item/clothing/under/liaison_suit/galaxy_blue, "white", "The latest in ultrafashion. for those with a cool demeanor."),
```

Comment:
Not sure if this should be something available through regular means due to how disruptive it is visually. Seems better fit for an admin-only CL reward.

Reply:
max is normally 50, these cost 51. The briefcase needs to be var-edited to have max_points = 51 for them to even be available. See the screen shot in a comment I made four days ago.

Reply:
Me stupid thanks Hugh